client/index.html

```diff
@@ -270,8 +270,8 @@ <h1 class="title">Settings</h1>
 
 							<div class="col-sm-12">
 								<label class="opt">
-									<input type="checkbox" name="notifyAllMessages">
-									Enable notification for all messages
+									<label for="old_password_input" class="sr-only">Custom Highlights (separate by ",")</label>
+									<input type="text" id="highlights" name="highlights" class="input" placeholder="Custom Highlights (separate by ',')">
```

Comment:
I think this was said on the channel but reminder to re-add the all-message notifications.